python/models.md

```diff
@@ -1,9 +1,31 @@
 # Model style
 
+General form:
+
+```
+class StyleGuide(models.Model):
+
```

Comment:
I just think it looks nicer.  The only time I don't have the space is if I'm defining a class property:

```
class MyView(TemplateView):
    template_name = 'foo.html'
```

```
class MyModel(models.Model):

    username = models.CharField()
```